### PR TITLE
En 10372 elrond go core integration

### PR DESCRIPTION
**Pull request by bogdan-rosianu**

### CODE REVIEW

**Comment by miiu96** [comment]
Please make a PR with this integration also against branch **feat/new-indexer**

File: dataIndexer_test.go [hunk 1]
```diff
@@ -12,15 +12,15 @@ import (
 	"github.com/ElrondNetwork/elastic-indexer-go/disabled"
 	"github.com/ElrondNetwork/elastic-indexer-go/mock"
 	"github.com/ElrondNetwork/elastic-indexer-go/workItems"
+	core2 "github.com/ElrondNetwork/elrond-go-core/core"
```
Comment (iulianpascalau) [inline dataIndexer_test.go:15]: core2 & data2

File: elasticProcessor_test.go [hunk 2]
```diff
@@ -13,15 +13,15 @@ import (
...
-	"github.com/ElrondNetwork/elrond-go/core"
-	nodeData "github.com/ElrondNetwork/elrond-go/data"
-	dataBlock "github.com/ElrondNetwork/elrond-go/data/block"
-	"github.com/ElrondNetwork/elrond-go/data/indexer"
-	"github.com/ElrondNetwork/elrond-go/data/receipt"
-	"github.com/ElrondNetwork/elrond-go/data/rewardTx"
-	"github.com/ElrondNetwork/elrond-go/data/smartContractResult"
-	"github.com/ElrondNetwork/elrond-go/data/transaction"
-	"github.com/ElrondNetwork/elrond-go/testscommon"
+	core2 "github.com/ElrondNetwork/elrond-go-core/core"
```
Comment (iulianpascalau) [inline elasticProcessor_test.go:16]: core2

File: workItems/workItemBlock.go [hunk 3]
```diff
@@ -3,12 +3,12 @@ package workItems
 import (
 	"fmt"
 
+	"github.com/ElrondNetwork/elrond-go-core/core/check"
+	data2 "github.com/ElrondNetwork/elrond-go-core/data"
```
Comment (iulianpascalau) [inline workItems/workItemBlock.go:7]: data2

**Review by miiu96 — COMMENTED** [review]
good job

File: dataIndexer_test.go (hunk 1 above)
Comment (bogdan-rosianu) [reply dataIndexer_test.go]: fixed

File: elasticProcessor_test.go (hunk 2 above)
Comment (bogdan-rosianu) [reply elasticProcessor_test.go]: fixed

File: workItems/workItemBlock.go (hunk 3 above)
Comment (bogdan-rosianu) [reply workItems/workItemBlock.go]: fixed

**Review by iulianpascalau — APPROVED** [review]

**Review by miiu96 — APPROVED** [review]

**Pull request merged**